src/Installer/pkg/dotnet-sdk.proj

```diff
@@ -0,0 +1,160 @@
+    <PackageBrandNameSuffix>SDK</PackageBrandNameSuffix>
+    <!--
+      Installer infra requires MajorVersion and MinorVersion, which triggers reevaluation of Arcade's
+      VersionPrefix property. This requires PatchVersion to be set in order to produce the correct SDK version.
+    -->
+    <MajorVersion>$(VersionMajor)</MajorVersion>
+    <MinorVersion>$(VersionMinor)</MinorVersion>
+    <PatchVersion>$(VersionPrefix.Split('.')[2])</PatchVersion>
+    <!--
+      All supported installer runtime identifiers should be specified here.
+      New arcade infra is only used for Linux installers, at the moment.
+    -->
+    <InstallerRuntimeIdentifiers>linux-$(Architecture)</InstallerRuntimeIdentifiers>
+    <IsShippingPackage>true</IsShippingPackage>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\redist-installer\redist-installer.csproj" ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Installers" />
+  </ItemGroup>
+
+  <Import Project="$(RepoRoot)src\Tasks\sdk-tasks\sdk-tasks.InTree.targets" />
+
+  <PropertyGroup>
+    <RedistLayoutPath>$(ArtifactsBinDir)redist-installer\$(Configuration)\dotnet\</RedistLayoutPath>
+    <ManpagesDirectory>$(RepoRoot)documentation/manpages/sdk</ManpagesDirectory>
+    <CLISdkRoot>$(RedistLayoutPath)sdk/</CLISdkRoot>
+    <TemplatesRoot>$(RedistLayoutPath)templates/</TemplatesRoot>
+  </PropertyGroup>
+
+  <Target Name="PublishToDisk">
+    <Error Condition="'$(OutputPath)' == ''" Text="Publishing to disk requires the OutputPath to be set to the root of the path to write to." />
+
+    <ItemGroup>
+      <CLISdkFiles Include="$(CLISdkRoot)**/*" />
+      <TemplatesFiles Include="$(TemplatesRoot)**/*" />
+      <ManifestFiles Include="$(RedistLayoutPath)sdk-manifests/**/*" />
+    </ItemGroup>
+
+    <!-- Create layout: Binaries -->
+    <Copy
+      DestinationFiles="@(CLISdkFiles->'$(OutputPath)/sdk/%(RecursiveDir)%(Filename)%(Extension)')"
+      SourceFiles="@(CLISdkFiles)"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+
+    <!-- Create layout: Templates -->
+    <Copy
+      DestinationFiles="@(TemplatesFiles->'$(OutputPath)/templates/%(RecursiveDir)%(Filename)%(Extension)')"
+      SourceFiles="@(TemplatesFiles)"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+
+    <!-- Create layout: Workload Manifests -->
+    <Copy
+      DestinationFiles="@(ManifestFiles->'$(OutputPath)/sdk-manifests/%(RecursiveDir)%(Filename)%(Extension)')"
+      SourceFiles="@(ManifestFiles)"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+  </Target>
+
+  <Target Name="SetCustomPackagingProperties" AfterTargets="_GetInstallerProperties">
+    <PropertyGroup>
+      <PackageSummary>Microsoft .NET SDK $(PackageVersion)</PackageSummary>
```

Comment:
If you set `<UseBrandingNameInLinuxPackageDescription>true</UseBrandingNameInLinuxPackageDescription>` in the property group on the top of the project, you'll get a package summary like `Microsoft .NET SDK - 10.0.101` instead of `Microsoft .NET SDK 10.0.101` and (in combination with https://github.com/dotnet/arcade/pull/15308) not need this custom target.

Reply:
Hmm, it's worth waiting for arcade to flow to sdk, to be able to utilize that change and get rid of this target.

Or we can do this as a follow-up.

Reply:
I'm fine with either waiting or doing this as a follow-up cleanup.